db/seeds/dummy_structures.rb

```diff
@@ -106,6 +106,7 @@
 if Rails.env.deployed_development?
   EngageAndLearnApiToken.find_or_create_by!(hashed_token: "dfce9a34c6f982e8adb4b903f8b6064682e6ad1f7858c41ed8a0a7468abc8896")
   NpqRegistrationApiToken.find_or_create_by!(hashed_token: "1dae3836ed90df4b796eff1f4a4713247ac5bc8a00352ea46eee621d74cd4fcf")
+  DataStudioApiToken.find_or_create_by!(hashed_token: "c7123fb0e2aecb17e1089e01849d71665983e200e891fe726341a08f176c1d64")
```

Comment:
Just to make sure - how did you get this hashed token exactly?

I would expect the answer to be 'I ssh-ed into review app, made a new token, and got the hashed version of that from there'. 

If you got it from a local version of your app, it won't work - cause the base that devise uses to digest tokens is different between dev and deployed dev (I think).

Reply:
I ssh'd into dev and did it there, it appears to work.